src/pytorch_lightning/strategies/deepspeed.py

```diff
@@ -359,6 +360,7 @@ def setup_distributed(self):
     def setup(self, trainer: "pl.Trainer") -> None:
         self.accelerator.setup(trainer)
         # we set the device so that optimizers can be created with distributed comms.
+        assert isinstance(self.lightning_module, pl.LightningModule)
```

Comment:
```suggestion
        assert self.lightning_module is not None
```
I think it should check the other case.